test/test_quantized.py

```diff
@@ -7,12 +7,11 @@
 import torch.jit
 import torch.nn.functional as F
 
-from hypothesis import assume, given
+import hypothesis as hy
```

Comment:
I think it's common to do it in original way, why do we want to change?

Reply:
I was afraid of the name conflicts :/